packages/terra-date-picker/src/DatePicker.module.scss

```diff
@@ -81,9 +81,9 @@
     }
   }
 
-  .date-input-month,
-  .date-input-day,
-  .date-input-year {
+  .date-input .date-input-month,
+  .date-input .date-input-day,
+  .date-input .date-input-year {
```

Comment:
This change was to deal with an issue where terra-form-input css would overwrite this css due to specificity rules. This seems to work but I'd happy with second opinions.
@tbiethman @benbcai

Reply:
Does it work if we just move these inside the .date-input class on line 144 so that we don't need to add it to each one of these?